foundry.toml

```diff
@@ -3,7 +3,7 @@ src = "src"
 out = "out"
 libs = ["lib", "dependencies"]
 optimizer = true
-optimizer_runs = 200
+optimizer_runs = 10
```

Comment:
how come we keep bringing this down? I’d have assumed more == better

Reply:
It reduces the deployment bytecode size. The contract size exceeded the deployment limit of 24,576 bytes.
So had to do some gas optimisation within the code and also reduce the optimizer runs to get it to deploy. @CluEleSsUK